flink-table/flink-table-common/src/test/java/org/apache/flink/table/types/inference/InputTypeStrategiesTestBase.java

```diff
@@ -45,39 +43,39 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /** Base class for testing {@link InputTypeStrategy}. */
-@RunWith(Parameterized.class)
 public abstract class InputTypeStrategiesTestBase {
 
-    @Parameterized.Parameter public TestSpec testSpec;
-
-    @Test
-    public void testStrategy() {
+    @ParameterizedTest
```

Comment:
```suggestion
```
Since the method is called from the sub-classes.

Although I'm curious; what would happen if this method had a `@MethodSource`, but the method only exists in sub-classes?

Reply:
What we could also try is using Lifecycle.PER_CLASS and an abstract `testData()` method that sub-classes implement.

Reply:
Thanks for the hint 
it seems working